src/plugins/intel_cpu/src/config.cpp

```diff
@@ -324,8 +342,7 @@ void Config::updateProperties() {
     IE_SUPPRESS_DEPRECATED_START
         _config.insert({ PluginConfigParams::KEY_DUMP_EXEC_GRAPH_AS_DOT, dumpToDot });
     IE_SUPPRESS_DEPRECATED_END;
-
-    if (enforceBF16) {
+    if (inferencePrecision == ov::element::bf16) {
         _config.insert({ PluginConfigParams::KEY_ENFORCE_BF16, PluginConfigParams::YES });
     } else {
```

Comment:
that means we also need to add a new config param called `ENFORCE_F16`, and it looks like that 
both `ENFORCE_BF16` & `ENFORCE_F16` are redundant and maybe conflict with another framework level config param `INFERENCE_PRECISION_HINT`,  and since `INFERENCE_PRECISION_HINT` is widely supported by all plugins, and `ENFORCE_xxx` config only appears in CPU plugin, so I choose not to add another CPU specific config